embuilder.py

```diff
@@ -246,6 +246,13 @@ def main():
           return int(emscripten_GetProcAddress("waka waka"));
         }
       ''', ['libgl-mt.bc'], ['-s', 'USE_PTHREADS=1'])
+    elif what == 'gl-webgl2':
+      build('''
+        extern "C" { extern void* emscripten_GetProcAddress(const char *x); }
+        int main() {
+          return int(emscripten_GetProcAddress("waka waka"));
+        }
+      ''', ['libgl-webgl2.bc'], ['-s', 'USE_WEBGL2=1'])
```

Comment:
all additions to embuilder should be added to `test_embuilder`, and to either `USER_TASKS` or `SYSTEM_TASKS` in the embuilder file.

Reply:
What is the role of that test? The CI `build` step already does `python embuilder.py build ALL`, which builds all the variants of `gl`, and verifies that the individual generated .bc files exist?

Reply:
Added the different variants in the build

Reply:
The test verifies the specific files that are generated, and guards against "ALL" not being complete. But yeah, it's not very clever or in-depth or anything like that.